doc/start/documenting-ceph.rst

```diff
@@ -397,8 +397,8 @@ the packages. Then, download them from a mirror and install them. For example:
 	wget http://rpmfind.net/linux/centos/7/os/x86_64/Packages/python-sphinx-1.1.3-11.el7.noarch.rpm
 	sudo yum install python-sphinx-1.1.3-11.el7.noarch.rpm
 
-Ceph documentation makes extensive use of `ditaa`_, which is not presently built
-for CentOS/RHEL7. You must install ``ditaa`` if you are making changes to
+Ceph documentation makes extensive use of `ditaa`_, which is not currently
+built for CentOS/RHEL7. You must install ``ditaa`` if you are making changes to
```

Comment:
We don't support EL7 at all now, so I might modernize the whole section with the assumption that 8 or ideally 9 is the minimum in-scope.

Reply:
I might just remove all references to CentOS and RHEL, since the carnage of six years ago.